app/src/main/java/org/mozilla/focus/MainActivity.kt

```diff
@@ -115,6 +119,21 @@ class MainActivity : LocaleAwareAppCompatActivity(), BrowserFragmentCallbacks, U
             }
         }
 
+        val bannerLayout: View? = findViewById(R.id.bannerLayout)
```

Comment:
Since I don't know this app super well, I've chosen a more defensive way to include this UI where I specifically check for views to exist. I assume this is not really needed but it gives me a little extra confidence.